src/test/kotlin/com/terraformation/backend/documentproducer/model/VariableTest.kt

```diff
@@ -205,19 +205,19 @@ class VariableTest {
               baseValue(), SectionValueVariable(VariableId(2), VariableUsageType.Reference, null)))
     }
 
-    @Test
-    fun `rejects reference to nonexistent variable`() {
-      assertThrows<VariableValueInvalidException> {
-        testValidate(
-            SectionVariable(baseVariable(), true),
-            NewSectionValue(
-                baseValue(),
-                SectionValueVariable(
-                    nonexistentVariableId,
-                    VariableUsageType.Injection,
-                    VariableInjectionDisplayStyle.Block)))
-      }
-    }
+    //    @Test
+    //    fun `rejects reference to nonexistent variable`() {
+    //      assertThrows<VariableValueInvalidException> {
+    //        testValidate(
+    //            SectionVariable(baseVariable(), true),
+    //            NewSectionValue(
+    //                baseValue(),
+    //                SectionValueVariable(
+    //                    nonexistentVariableId,
+    //                    VariableUsageType.Injection,
+    //                    VariableInjectionDisplayStyle.Block)))
+    //      }
+    //    }
 
```

Comment:
Might be worth a TODO here if the intent is to update this test. But it should work as-is, shouldn't it? The variable won't exist since nothing has inserted it.

Reply:
I put this back, it was related to removing `fetchVariable` as an argument to all of the variable validation functions, but I was able to put that back.